src/doc/yaml/cms.yaml

```diff
@@ -0,0 +1,567 @@
+              - familyname
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Read successful
+          $ref: '#/components/responses/CredentialRequestResponse'
+        '400':
+          description: Bad request
+        '403':
+          description: Read not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    patch:
+      tags:
+        - Credential Request
+      summary: Update a credential request
+      operationId: updateCredentialRequest
+      parameters:
+        - name: credentialRequestId
+          in: path
+          description: the id of the credential request
+          required: true
+          schema:
+            type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CredentialRequest'
+      responses:
+        '204':
+          description: Update successful
+        '400':
+          description: Bad request
+        '403':
+          description: Update not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    delete:
+      tags:
+        - Credential Request
+      summary: Delete a credential request
+      operationId: deleteCredentialRequest
+      parameters:
+        - name: credentialRequestId
+          in: path
+          description: the id of the credential request
+          required: true
+          schema:
+            type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Delete successful
+        '400':
+          description: Bad request
+        '403':
+          description: Delete not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /v1/credential:
+    get:
+      tags:
+        - Credential
+      summary: Retrieve a list of credentials that match the passed in search criteria
+      operationId: findCredentials
+      parameters:
+        - name: expressions
+          in: query
```

Comment:
I believe it should be in the body and not as a query. It will be difficult to express complex expressions using only query parameters. (should also be a POST if there is a body)